libbpf-tools/bits.bpf.h

```diff
@@ -25,4 +25,16 @@ static __always_inline u64 log2l(u64 v)
 		return log2(v);
 }
 
+static __always_inline
+bool str_filtered(const char *target, const volatile char *cond, int cond_len)
```

Comment:
this is not bit manipulation thing, it's also not complicated and tricky at all... so I don't know, it definitely doesn't belong to bits.bpf.h, but creating another header just for this also seems like a waste

Reply:
Oh, I think I misunderstood the `bit-manipulation`, your meaning is `bit operation`?  I misunderstood it as light(bit) helpers. I’ll change it back. Thanks.

Reply:
Fixed.